### PR TITLE
update workflow component to take any child component as prop

### PR DESCRIPTION
**Pull request by iamvigneshwars**

### CODE REVIEW

File: frontend/workflows-lib/lib/components/workflow/WorkflowAccordian.tsx [hunk 1]
```diff
@@ -15,7 +15,10 @@ const AccordionDetails = styled(MuiAccordionDetails)(({ theme }) => ({
   height: "80vh",
 }));
 
-const WorkflowAccordion: React.FC<WorkflowProps> = ({ workflow }) => {
+const WorkflowAccordion: React.FC<WorkflowProps> = ({
+  workflow,
+  ChildComponent,
+}) => {
```
Comment (garryod) [inline frontend/workflows-lib/lib/components/workflow/WorkflowAccordian.tsx:21]: Is there a reason we're passing it like this an not [as `children`](https://react.dev/learn/passing-props-to-a-component#passing-jsx-as-children)?

**Review by garryod — COMMENTED** [review]
Looking good, just one question on `children`

File: frontend/workflows-lib/lib/components/workflow/WorkflowAccordian.tsx (hunk 1 above)
Comment (iamvigneshwars) [reply frontend/workflows-lib/lib/components/workflow/WorkflowAccordian.tsx:21]: We're doing the same thing, we just have another prop for the workflow name and status.

Comment (garryod) [reply frontend/workflows-lib/lib/components/workflow/WorkflowAccordian.tsx:21]: Yeah, but if we were to call it `children` then we could use the normal JSX syntax of:
```javascript
<WorkflowAccordion>
    <Tasks/>
</WorkflowAccordion>
```

Comment (iamvigneshwars) [reply frontend/workflows-lib/lib/components/workflow/WorkflowAccordian.tsx]: Ahh got it, fixed it!

**Pull request merged**